src/Scrutor/ServiceCollectionExtensions.Scanning.cs

```diff
@@ -32,6 +32,25 @@ public static IServiceCollection Scan(this IServiceCollection services, Action<I
             return services.Populate(selector, RegistrationStrategy.Append);
         }
 
+        public static IServiceCollection Scan(this IServiceCollection services, Action<ITypeSelector> action)
```

Comment:
Ah, so it's just inferred from usage?

> would you prefer to have a different name than Scan ?

Nah, I think it's fine. It doesn't really matter to the user which type is used. It's much better UX to have a single point of entry for the scanning :smile: